src/task/script.js

```diff
@@ -387,20 +388,30 @@ function clearUserNotFound() {
 function createSuggestionsList(matches) {
   const listItems = document.getElementById('list-items');
   if (matches.length) {
-    matches.map(({ username }) => {
+    matches.map(({ username, picture = {} }) => {
+      const defaultImageUrl =
+        'https://members.realdevsquad.com/images/Avatar.png?90.2370159455815';
```

Comment:
Will the URL always remain the same?
It would good to create a images folder at /src and place this image there

Reply:
yes, the default avatar will be going to be the same, I thought of putting the image in a folder but wasn't sure which avatar image I should be using here, so, for now, the one is getting used on the member's page, I used it here. Should I put this one in an image folder, or do I have to use a different avatar?

Reply:
This avatar looks good.
What if for some reason URL is changed,  the image gets updated, members site goes down, etc I think it would be better if we place this image in the dashboard site repo itself since this will remove our dependency on the member's site

Reply:
make sense, will place the image in a folder